frontstage/controllers/party_controller.py

```diff
@@ -461,6 +461,7 @@ def get_survey_list_details_for_party(respondent: dict, tag: str, business_party
                 "formatted_submit_by": collection_exercise["events"]["return_by"]["formatted_date"],
                 "due_in": collection_exercise["events"]["return_by"]["due_time"],
                 "collection_exercise_ref": collection_exercise["exerciseRef"],
+                "collection_exercise_id": collection_exercise["id"],
```

Comment:
If/when we deliver this for real it might be the exerciseRef we want to capture. The intended audience for these metrics would be the business and as such the UUID might not be appropriate.

Reply:
Done